react/react1/week2/todo-app/src/App.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+
+import Header from './components/Header';
+import Counter from './components/Counter';
+import ListTodos from './components/ListTodos';
+
+export default function App() {
+  const [todos, setTodos] = useState([
+    {
+      id: 1,
+      description: 'Get out of bed',
+    },
+    {
+      id: 2,
+      description: 'Brush teeth',
+    },
+    {
+      id: 3,
+      description: 'Eat breakfast',
+    },
+  ]);
+
+  const handleDelete = (id) => {
+    const updatedTodos = todos.filter((todo) => todo.id !== id);
+    setTodos(updatedTodos);
+  };
+
+  return (
+    <div className="app-container">
+      <Header title={'Todo List'} />
+      <Counter />
+      <button
+        className="btn-add"
+        onClick={() =>
+          setTodos([
+            ...todos,
+            {
+              id: todos.length ? todos[todos.length - 1].id + 1 : 1,
```

Comment:
For now it is okay with the id, but if 2nd of the 4 todos is deleted before adding another todo, with your formula it takes id as 4 for the 5th todo and that gives you an error.
So try research about unique id's for like `nanoid`.

Reply:
Thank you for the review!

I have been trying what you said and I am not getting any errors in my code.
The implementation of the id logic is that it will always create a new id, based on the last todo in the array +1. So if the last todo in the array has the id, 9 the new todo will have the id 10. 

I didn't want to add any libraries to this homework, I wanted to see if I can create a simple implementation without using Math.floor(Math.random()).
I will start using a library like `nanoid` in the future.

Reply:
Well, that is good 👍